src/js/services/TrackingServices.js

```diff
@@ -29,14 +29,28 @@ angular.module('blocktrail.wallet')
             ANALYTICS_META[eventVal] = { category: "BuyBTC" };
         });
 
+        var getBrowserFingerprint = function() {
+            // Return a promise for a fingerprint and its components
+            return new Promise(function (resolve, reject) {
+                new Fingerprint2().get(function (result, components) {
```

Comment:
should `{excludeFlashFonts: true}` as options since we don't make the flash file available anyway